library/ui-styles/src/main/res/values/styles_toolbar.xml

```diff
@@ -24,14 +26,16 @@
     <!-- main text -->
     <style name="Widget.Vector.TextView.ActionBarTitle" parent="TextAppearance.AppCompat.Widget.ActionBar.Title">
         <item name="android:textColor">?vctr_content_primary</item>
-        <item name="android:fontFamily">"sans-serif-medium"</item>
-        <item name="android:textSize">20sp</item>
+        <item name="android:fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">sans-serif-medium</item>
+        <item name="android:textSize">18sp</item>
     </style>
 
     <!-- sub text -->
     <style name="Widget.Vector.TextView.ActionBarSubTitle" parent="TextAppearance.AppCompat.Widget.ActionBar.Subtitle">
-        <item name="android:textColor">?vctr_content_primary</item>
-        <item name="android:fontFamily">"sans-serif-medium"</item>
+        <item name="android:textColor">?vctr_content_secondary</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="fontFamily">sans-serif</item>
```

Comment:
There is an existing naming problem with those 2 styles. As they are TextAppearance, their name should start with `TextAppearance.Vector` and not `Widget.Vector.TextView`. Can you take the opportunity to rename the 2 styles please?
The two new names would be:
- `TextAppearance.Vector.Widget.ActionBar.Title`
- `TextAppearance.Vector.Widget.ActionBar.Subtitle`

(I am not sure on the naming convention though. @ganfra WDYT?)